jetty-server/src/main/java/org/eclipse/jetty/server/HttpOutput.java

```diff
@@ -1670,15 +1696,6 @@ protected Action process() throws Exception
                 return Action.SCHEDULED;
             }
 
-            // all content written, but if we have not yet signal completion, we
-            // need to do so
-            if (_last && !_completed)
-            {
-                _completed = true;
-                channelWrite(BufferUtil.EMPTY_BUFFER, true, this);
-                return Action.SCHEDULED;
-            }
-
```

Comment:
Why was this removed?

Reply:
Because is can never ever happen.  I tried to write a unit test to give it code coverage and I was unable to do so.